jest.config.js

```diff
@@ -5,8 +5,8 @@ module.exports = {
     '\\.(mjs|js)$': 'babel-jest',
   },
   transformIgnorePatterns: [
-    'node_modules/!(@energyweb/ekc)',
```

Comment:
The [documentation](https://jestjs.io/docs/configuration#transformignorepatterns-arraystring) for `transformIgnorePatterns` says 
`An array of regexp pattern strings that are matched against all source file paths before transformation. If the file path matches any of the patterns, it will not be transformed.`

So I think the behaviour that I would expect here would be that only .mjs files in @energyweb/ewc would be transformed. Is this what we're looking for?

Maybe not worth thinking about too much as long as the tests are passing as I suppose this only affects the tests 😄

Reply:
I assume that all.mjs in any node_modules and all in @energyweb/ekc will be ignored, it is enaugh to match any of the pattern